web-registry/src/lib/db/types/json-ld/project-metadata.ts

```diff
@@ -1,37 +1,44 @@
 import { TypeValue, UrlList, UrlType } from 'lib/rdf/types';
 
+/** Anchored metadata AKA "Additional Info" - Editable only with a signed Ledger TX. */
 export interface ProjectMetadataLD {
```

Comment:
one question i have is where is the `vcsProjectId` for the C01 projects? i.e. https://github.com/regen-network/regen-registry-standards/blob/main/jsonld/projects/C01-verified-carbon-standard-project.jsonld#L11

i'm wondering whether i should put this in the `C01-Project` or `C01-Project-Page` schema, or do i need to think about it differently?

Reply:
as i'm working on changes in the registry standards, it seems like it might be helpful to have a project identifier (i.e. `vcsProjectId`) in both the `C01-Project` and the `C01-Project-Page` metadata

Reply:
agreed

Reply:
added these as optionals in `ProjectPageMetadataLD`